device-services/device-random/Dockerfile.build

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
This file not needed for examples since now CI on examples.

Reply:
Removed.